crates/oxc_linter/src/snapshots/consistent_test_it.snap

```diff
@@ -0,0 +1,455 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 216
+---
+  ⚠ eslint-plugin-jest(consistent-test-it): Enforce `test` and `it` usage conventions
+   ╭─[consistent_test_it.tsx:1:1]
+ 1 │ it("foo")
+   · ──
+   ╰────
+  help: Prefer using '"test"' instead of '"it"'
```

Comment:
Is it `help: Prefer using "test" instead of "it"` ?

Reply:
Done